docs/src/main/asciidoc/spring-cloud-sleuth.adoc

```diff
@@ -201,6 +201,125 @@ include::../../../../spring-cloud-sleuth-core/src/test/java/org/springframework/
 
 will lead in creating a span named `calculateTax`.
 
+== Managing spans with annotations
+
+=== Creating new spans
+
+If you really don't want to take care of creating local spans manually you can profit from the
+`@NewSpan` annotation. Also we give you the `@SpanTag` annotation to add tags in an automated
+fashion.
+
+Let's look at some examples of usage.
+
+[source,java]
+----
+include::../../../../spring-cloud-sleuth-core/src/test/java/org/springframework/cloud/sleuth/annotation/SleuthSpanCreatorAspectTests.java[tags=annotated_method,indent=0]
+----
+
+Annotating the method without any parameter will lead to a creation of a new span whose name
+will be equal to annotated method name.
+
+[source,java]
+----
+include::../../../../spring-cloud-sleuth-core/src/test/java/org/springframework/cloud/sleuth/annotation/SleuthSpanCreatorAspectTests.java[tags=custom_name_on_annotated_method,indent=0]
```

Comment:
I notice that "View" doesn't render this in github. maybe should be a github issue as otherwise we need to assume it works..